packages/synapse-sdk/src/synapse.ts

```diff
@@ -221,6 +247,23 @@ export class Synapse {
     return this._network
   }
 
+  /**
+   * Gets the telemetry service for error tracking and debugging
+   * @returns The telemetry service instance
+   * @example
+   * ```typescript
+   * // Get debug dump for support tickets
+   * const dump = synapse.telemetry.debugDump()
+   * console.log(JSON.stringify(dump, null, 2))
+   *
+   * // Track custom events
+   * synapse.telemetry.captureCustomEvent('user-action', { action: 'upload' })
+   * ```
+   */
+  get telemetry(): TelemetryService {
+    return this._telemetry
+  }
```

Comment:
We have two access patterns for the TelemetryService singleton: via synapse.telemetry and via the telemetry.singleton.  How about we just have one?

I assume the easiest is to move the construction of the TelemetryService into the singleton (passing it the telemetry config and context that are generated here) and then having everyone access via the singleton class?